temp/OnnxTransformTempSolution/Microsoft.ML.Transforms.Onnx/Program.cs

```diff
@@ -0,0 +1,79 @@
+                var dataIdv = ComponentCreation.CreateDataView(env, data);
+
+                
+                var modelFile = @"F:\ProjectsData\Sonoma\PoC\mlnet_dummy\OnnxTransformTempSolution\OnnxModel\lotusrt_cntk_fullyconnect.onnx";
+                var modelMetadata = new OnnxModelInfo()
+                {
+                    InputsInfo = new OnnxNodeInfo[]
+                    {
+                        new OnnxNodeInfo
+                        {
+                            Name = "input0",
+                            Shape = new OnnxShape { 1, 784 },
+                        }
+                    },
+
+                    OutputsInfo = new OnnxNodeInfo[]
+                    {
+                        new OnnxNodeInfo
+                        {
+                            Name = "output0",
+                            Shape = new OnnxShape { 1, 10 },
+                        }
+                    },
+                };
+
+                var transformArgs = new OnnxTransform.Arguments() { ModelFile = modelFile, InputColumn = "pixels", OutputColumn = "pixelsOut", ModelInfo = modelMetadata };
```

Comment:
>pixels [](start = 106, length = 6)

Doesn't this name need to match the name in modelMetadata?

Reply:
The assumption is that there's only one InputIdvColumn and the model has one TensorInputNode. Same with output. So the names don't have to match, b/c there's 1-1 mapping. 
For supporting multiple input/output, we need to do something similar to TF. Either require names to match, or allow users to create a mapping.